### PR TITLE
[LLVM10] EventFilter fix warnings for clang 10

### PR DESCRIPTION
**Pull request by mrodozov**
#### PR description:

Fixes warnings in clang 10

#### PR validation:

Builds without warnings with CLANG IB
tokenArraySeparator and tokenArrayEnd belong to the same enum so thats probably or not and

### CODE REVIEW

**Comment by mrodozov** [comment]
please test

File: EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc [hunk 1]
```diff
@@ -95,7 +95,7 @@ void RPCTwinMuxDigiToRaw::produce(edm::Event& event, edm::EventSetup const& setu
 
   std::map<int, FEDRawData> fed_data;
   // Loop over the FEDs
-  for (std::pair<int, std::vector<RPCAMCLink> > const& fed_amcs : fed_amcs_) {
+  for (std::pair<int, std::vector<RPCAMCLink> > const fed_amcs : fed_amcs_) {
```
Comment (slava77) [inline EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc:98]: this is strange.
why is a simple const OK here?  Isn't this going to trigger a copy of a vector?

Comment (mrodozov) [reply EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc:98]: Most of this fixes are suggested by the compiler. It says you need copy here or your need a reference here, in this case it said remove the reference
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc7_amd64_gcc820/CMSSW_11_1_CLANG_X_2020-04-27-2300/EventFilter/RPCRawToDigi

Comment (smuzaffar) [reply EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc:98]: @mrodozov , the correct fux is to use `std::pair<const int`

Comment (smuzaffar) [reply EventFilter/RPCRawToDigi/plugins/RPCTwinMuxDigiToRaw.cc:98]: @mrodozov , please remove this fix. It is already part of https://github.com/cms-sw/cmssw/pull/29572/files#diff-d409478036411a5bded20820a924b87a

**Comment by mrodozov** [comment]
please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/29592	cb69486 
- technical
- jenkins tests pass

**Comment by silviodonato** [comment]
@emeschi @mommsen
Could you sign or comment this PR by today, please? Today we will close CMSSW_11_1_0_pre7. Thanks!

**Comment by mommsen** [comment]
+1

Sorry for the delay in signing this PR.

**Comment by silviodonato** [comment]
+1

**Pull request merged**